src/libraries/System.Drawing.Common/src/System/Drawing/ImageAnimator.cs

```diff
@@ -381,10 +382,20 @@ public static void StopAnimate(Image image, EventHandler onFrameChangedHandler)
         private static void AnimateImages50ms()
         {
             Debug.Assert(s_imageInfoList != null, "Null images list");
+            DateTime animationStart = DateTime.Now;
```

Comment:
>Now [](http://example.com/codeflow?start=47&length=3)

using DateTime.UtcNow would be faster here as I don't think you need the local time anyway.